### ISSUE TITLE
SDK4.0-beta1: substituter.sh: line 19: [: !=: unary operator expected

### ISSUE THREAD

**Issue by tom-vandepoele** [Alfresco/alfresco-sdk#554]
## I'm submitting a ...   (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

## Current Behavior
Error during startup of share:
`/usr/local/tomcat/shared/classes/alfresco/substituter.sh: line 19: [: !=: unary operator expected`

## Possible Solution
add quotes to line 19 of substituter.sh
`if [ $CSRF_FILTER_REFERER != "" ] && [   $CSRF_FILTER_ORIGIN != "" ]; then`
to
`if [ "$CSRF_FILTER_REFERER" != "" ] && [   "$CSRF_FILTER_ORIGIN" != "" ]; then`

## Steps to Reproduce (for bugs)
Startup project with SDK4.0

## Your Environment
* Alfresco SDK version used: 4.0.0-beta-1
* Alfresco version used: 6.0
* Windows development machine

## Context

share-config-custom.xml not being properly updated with CSRF filter policy

## Workaround
Basically copy a good version of this file into the docker image:

- copy out the file after startup with:
`docker cp docker_<project>-share_1:/usr/local/tomcat/shared/classes/alfresco/substituter.sh .`
- Put the file in src/main/docker of share-docker project, modify it as mentioned above.
- Modify src/main/docker/Dockerfile
`COPY substituter.sh /usr/local/tomcat/shared/classes/alfresco/substituter.sh`

**Comment by ohej** [comment]
Hi Tom

This really should be fixed upstream in the Share docker image, rather than from within the SDK. 

Can I ask that you open an issue on https://github.com/Alfresco/share ?

**Comment by ohej** [comment]
Essentially it should be fixed here: https://github.com/Alfresco/share/blob/master/packaging/docker/substituter.sh#L19
